src/browser/monkey-menu.html

```diff
@@ -40,66 +41,82 @@
       <div class="text heading">Other user scripts</div>
     </div>
 
-    <div rv-each-script="userScripts.inactive" class="menu-item user-script"
+    <a rv-each-script="userScripts.inactive"
+        rv-href="script.uuid | mmUuidMenu"
+        rv-data-uuid="script.uuid"
         rv-class-disabled="script.enabled | not"
-        rv-data-user-script-uuid="script.uuid"
-    >
+        class="menu-item user-script">
       <div class="icon fa-fw"><img rv-src="script.icon"></div>
       <div class="text">{script.name}</div>
-    </div>
+    </a>
   </div>
 
   <hr>
 
-  <div class="menu-item" id="new-user-script">
+  <a href="#new-user-script"
+      class="menu-item">
     <div class="icon fa fa-fw fa-file-text-o"></div>
     <div class="text">New user script ...</div>
-  </div>
+  </a>
 
-  <div class="menu-item" data-url="http://www.greasespot.net/">
+  <a href="#open-tab_https://www.greasespot.net/"
+      class="menu-item">
     <div class="icon fa fa-fw fa-link"></div>
     <div class="text">Greasemonkey home page</div>
-  </div>
+  </a>
 
-  <div class="menu-item" data-url="https://wiki.greasespot.net/">
+  <a href="#open-tab_https://wiki.greasespot.net/"
+      class="menu-item">
     <div class="icon fa fa-fw fa-link"></div>
     <div class="text">Greasemonkey Wiki</div>
-  </div>
+  </a>
 
-  <div class="menu-item"
-      data-url="https://wiki.greasespot.net/User_Script_Hosting">
+  <a href="#open-tab_https://wiki.greasespot.net/User_Script_Hosting"
```

Comment:
I'm not excited about making up custom string-concatenated formatting of various values that have to be carefully parsed back out again.  I think there's a better way to do this.  Especially in this case, we need to have a JS handler set to avoid navigating to this fake anchor anyway.  So just put the real URL, cancel the event, and do the create-a-tab API call with the actual HREF value.